.github/workflows/ci.yml

```diff
@@ -107,6 +107,25 @@ jobs:
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
 
+  self-hosted-runners:
```

Comment:
Make the name shorter (`our-runners` or `self-hosted`) so the os from the matrix is not truncated in the GitHub UI.